projects/storefrontapp-e2e-cypress/cypress/integration/cx_ccv2/regression/b2b/b2b-checkout.ts

```diff
@@ -0,0 +1,475 @@
+import { tabbingOrderConfig as config } from '../../../../helpers/accessibility/b2b/tabbing-order.config';
+import {
+  b2bProduct,
+  b2bUser,
+  order_type,
+  poNumber,
+  POWERTOOLS_BASESITE,
+  products,
+  recurrencePeriod,
+  recurrencePeriodMap,
+  replenishmentDate,
+  replenishmentDay,
+} from '../../../../sample-data/b2b-checkout';
+
+import {
+  b2bAccountShipToUser,
+  b2bUnit,
+  costCenter,
+} from './ccv2-b2b-checkout-sample-data';
+
+import { verifyTabbingOrder } from '../../../../helpers/accessibility/tabbing-order';
+import {
+  addCheapProductToCart,
+  visitHomePage,
+  waitForPage,
+  waitForProductPage,
+} from '../../../../helpers/checkout-flow';
+import {
+  getSampleUser,
+  SampleCartProduct,
+  SampleProduct,
+  SampleUser,
+} from '../../../../sample-data/checkout-flow';
+import { myCompanyAdminUser } from '../../../../sample-data/shared-users';
+import { login } from '../../../../support/utils/login';
+
+export function loginB2bUser() {
+  let adminToken;
+  let user = getSampleUser();
+
+  login(
+    myCompanyAdminUser.registrationData.email,
```

Comment:
same for needing to login as admin to create a user

* a quick refreshment will help me